src/mongo_log.c

```diff
@@ -32,10 +33,18 @@ void mongolite_log_handler (mongoc_log_level_t event, const char *log_domain, co
 
 void R_init_mongolite(DllInfo *info) {
   static mongoc_log_func_t logfun = mongolite_log_handler;
+  char *r_version;
   mongoc_init();
+#if defined (R_VERSION)
+  r_version = bson_strdup_printf ("R=%s.%s ", R_MAJOR, R_MINOR);
+#else
+  r_version = bson_strdup ("R=undefined ");
+#endif
+  mongoc_handshake_data_append ("mongolite", "", r_version);
```

Comment:
The documentation for `mongoc_handshake_data_append` is in the libmongoc source in [mongoc-handshake.h](https://github.com/mongodb/mongo-c-driver/blob/3c6d9a8e6b11f4a830b6b7c72d9bc176b86585d1/src/libmongoc/src/mongoc/mongoc-handshake.h#L31-L91).

I left the second argument, `driver_version`, as an empty string. Is it possible to obtain the version of mongolite in the C headers? I.e. the version `2.3.1` from the DESCRIPTION file?